python/composio/client/__init__.py

```diff
@@ -436,42 +437,36 @@ def initiate_connection(
         :param integration: Optional existing IntegrationModel instance to be used.
         :return: A ConnectionRequestModel instance representing the initiated connection.
         """
+        if force_new_integration is not None:
+            warnings.warn(
+                "`force_new_integration` parameter has been deprecated "
+                "and will be removed in v0.9.0."
+            )
+
         app = self.client.apps.get(name=App(app_name).slug)
         timestamp = datetime.now().strftime("%Y%m%d%H%M%S")
-        if integration is None and auth_mode is not None:
-            if auth_mode not in AUTH_SCHEME_WITH_INITIATE:
-                raise InvalidParams(
-                    f"'auth_mode' should be one of {AUTH_SCHEME_WITH_INITIATE}"
-                )
 
-            auth_mode = t.cast(AuthSchemeType, auth_mode)
-            if "OAUTH" not in auth_mode:
-                use_composio_auth = False
-
-            integration = self.client.integrations.create(
-                app_id=app.appId,
-                name=f"{app_name}_{timestamp}",
-                auth_mode=auth_mode,
-                auth_config=auth_config,
-                use_composio_auth=use_composio_auth,
-                force_new_integration=force_new_integration,
+        if auth_mode is None:
+            raise InvalidParams(
+                f"'auth_mode' should be one of {AUTH_SCHEME_WITH_INITIATE}"
```

Comment:
This is very confusing message/check, if `auth_mode` is required why is an optional parameter?